pages/guides/how-to-create-tradingview-charts.mdx

```diff
@@ -4,7 +4,7 @@ import { Tab, Tabs } from "nextra-theme-docs";
 
 The TradingView integration is a web-based widget that allows users to view Pyth prices on their own website. All Pyth prices made available through the TradingView integration are originating from [Pythnet](pythnet-price-feeds.md).
 
-## Technical Integration
```

Comment:
I suggest explaining in a 2nd paragraph in the info that there are 2 ways to integrate with tradingview, and give the rough tradeoffs between the two there.